Source/Engine/PackageBuilder/SyntaxParser.cs

```diff
@@ -96,9 +96,12 @@ public PackageSyntax ParseExpressionText(string text)
             fTextPosition = -1;
             NextCharacter();
             NextToken();
+            int startPosition = fToken.TextSlice.Position;
             Syntax patternBody = ParseInsideOrOutsideOrHaving();
             PatternSyntax pattern = Syntax.Pattern(isSearchTarget: true, "Pattern", patternBody, null);
+            pattern = SetTextRange(pattern, startPosition);
```

Comment:
Do not assign return value of SetTextRange into a variable if there is no need for it. Returning updated value from SetTextRange is just for scenarios like `return` to avoid adding extra line.